src/site/markdown/server/manage-webadmin.md

```diff
@@ -1595,13 +1595,16 @@ Response codes:
 curl -XDELETE http://ip:port/mailQueues/mailQueueName/mails?sender=senderMailAddress
 ```
 
+##### For deleting emails in a mail queue by conditions:
 This request should have exactly one query parameter from the following list:
 
 * sender: which is a mail address (i.e. sender@james.org)
 * name: which is a string
 * recipient: which is a mail address (i.e. recipient@james.org)
 
 The mails from the given mail queue matching the query parameter will be deleted.
+##### For deleting all emails in a mail queue:
+* Pass no parameter in the url
```

Comment:
I'm pretty sure the type of task created is different...

Reply:
Also I think you can improve formatting and wording here (Do we need  subsection at all?)